src/organizations/components/OrgProfileTab/DeletePaidOrgOverlay.tsx

```diff
@@ -115,6 +125,12 @@ export const DeletePaidOrgOverlay: FC = () => {
       .catch(err => {
         clearTimeout(responseTimer)
         dispatch(notify(deleteOrgFailed(SupportLink, org.name)))
+        event(DeleteOrgOverlay.DeleteOrgFail, multiOrgTag, {
+          userId: org.id,
+          orgName: org.name,
+          accountType: account.type,
+          accountId: account.id,
+        })
```

Comment:
Same comment here - I'm not sure we need to event org deletion failure, as we're already catching it through honeybadger, and the use of this information isn't analytics - how many org deletions failed - but rather, identifying a problem with the UI that needs to be resolved.

For the same reason we don't need to call the event an OrgDeleteSuccess - it can just be [eventName].OrgDeleted, or similar.